### PR TITLE
Add homepage frontend and assets

### PR DESCRIPTION
**Pull request by ERICCYS**
Create the home page and user-info page, add frontend code of the homepage.
The backend of the home page is not there yet. The data used in the frontend is dummy data.

### CODE REVIEW

**Comment by MaxPoon** [comment]
Please delete all the .DS_Store files and add `.DS_Store` to .gitignore. In the future it will be better to upload some screenshots to show how the pages look like.

File: client/app.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "pages":[
+    "pages/home/home",
```
Comment (MaxPoon) [inline client/app.json]: Should we create a new home page or just modify the existing index page?

**Review by MaxPoon — CHANGES_REQUESTED** [review]

File: client/app.json (hunk 1 above)
Comment (ERICCYS) [reply client/app.json]: I am not sure whether there is a convention just leave the index page there. If not I think we can just modify the index page.

Comment (MaxPoon) [reply client/app.json]: Please do it. It sounds more reasonable to me :)

Comment (ERICCYS) [reply client/app.json]: Okay, no problem!

Comment (ERICCYS) [reply client/app.json]: Oh, do I need to delete the unrelated page such as "addCgi" page?

Comment (MaxPoon) [reply client/app.json]: Yes please.

File: client/app.json [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
-    "navigationBarTitleText": "Wafer Quick Start",
+    "navigationBarBackgroundColor": "#73a96b",
+    "navigationBarTitleText": "TPCC YEC Soccer Team",
     "navigationBarTextStyle":"black"
+  },
+  "tabBar": {
+    "list": [
+      {
+        "pagePath": "pages/index/index",
+        "text": "通知",
```
Comment (MaxPoon) [inline client/app.json]: Please use English.

File: client/app.json (hunk 1 above)
Comment (ERICCYS) [reply client/app.json]: The page has been deleted, please check that.

Comment (MaxPoon) [reply client/app.json]: Although you've added `.DS_Store` to `.gitignore`, the files that you have already added won't be removed automatically. Please run `git rm *.DS_Store` and commit.

Comment (ERICCYS) [reply client/app.json]: Deleted, please check.

File: project.config.json [hunk 3]
```diff
@@ -7,6 +7,6 @@
 		"newFeature": true
 	},
 	"appid": "wx13a399684987dfa0",
-	"projectname": "TPCC-YEC-Soccer-Team-Mini-Program",
+	"projectname": "homepagem",
```
Comment (MaxPoon) [inline project.config.json]: Why do we need to change the project name?

File: client/app.json (hunk 2 above)
Comment (ERICCYS) [reply client/app.json]: Okay will change this

File: client/app.json [hunk 4]
```diff
@@ -1,13 +1,37 @@
 {
   "pages":[
     "pages/index/index",
-    "pages/addCgi/addCgi"
+    "pages/userinfo/userinfo",
+    "pages/photo/photo",
+    "pages/editntc/editntc"
```
Comment (MaxPoon) [inline client/app.json:6]: Good - it makes the following tasks slightly easier.

File: project.config.json (hunk 3 above)
Comment (ERICCYS) [reply project.config.json]: I think Xinyi changed this accidentally, will change it back

File: client/app.json (hunk 2 above)
Comment (ERICCYS) [reply client/app.json]: What word should we use? "Notification" or "Home" ?

Comment (MaxPoon) [reply client/app.json]: Both sound good to me. So please use "Home" as it's shorter :)

Comment (ERICCYS) [reply client/app.json]: Sure :>

Comment (ERICCYS) [reply client/app.json]: By the way, do we need to make the page name to be consistent with the name in the tabbar?
For example, now, the third page in the tabbar is called "Me" but the page name is "userinfo", do we need to change this?

Comment (MaxPoon) [reply client/app.json]: I think that's fine.

Comment (ERICCYS) [reply client/app.json]: Okay, I have made the changes, except the home page (still called index page)

Comment (ERICCYS) [reply client/app.json]: Please check that, thanks!

**Comment by MaxPoon** [comment]
A lot of files changed are included in this PR due to the changes of file mode from 644 to 755. I'm fine with it. But I think in future we all should set `git config core.filemode false` so that the changes of file mode won't be included in the PR. It'll make the reviewer's life easier.

**Comment by ERICCYS** [comment]
> A lot of files changed are included in this PR due to the changes of file mode from 644 to 755. I'm fine with it. But I think in future we all should set `git config core.filemode false` so that the changes of file mode won't be included in the PR. It'll make the reviewer's life easier.

Okay get it! But I am not very clear where did we change the file mode? For this point I am a bit confusing. Are there some files or commands we need to notice to not change the filemode? We will definetely note that in the later PR.

**Review by MaxPoon — APPROVED** [review]

**Pull request merged**